2.arrayOccurenceFilter.js

```diff
@@ -11,6 +11,11 @@
 // occurrenceFilter([1, 3, 4, 1, 9, 1, 3, 4, 3, 1, 2], 2) returns [1, 3, 4] 
 
 const occurrenceFilter = (arr, n) => {
+    let output = []
+    let count = n 
```

Comment:
counts should be an object. Please go back and revisit this problem. Creating counter objects is a very common pattern in many problems. A must learn, so please revist.